app/src/main/res/layout/content_main.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    app:layout_behavior="@string/appbar_scrolling_view_behavior"
+    tools:showIn="@layout/app_bar_main">
+
+    <fragment
```

Comment:
We should maybe use FragmentContainerView instead?

Reply:
For the moment, most of my fragment transactions are done through the DrawerMenu and I didn't had the need to use that class which gives help when building complex UI with many fragment transactions. But I will studiy it more in details, It could be useful later, thanks for the idea !